sync/class.jetpack-sync-actions.php

```diff
@@ -41,6 +46,11 @@ static function init() {
 
 		// On jetpack registration
 		add_action( 'jetpack_site_registered', array( __CLASS__, 'schedule_full_sync' ) );
+
+		// Schedule a job to send DB checksums once an hour
+		if (! wp_next_scheduled ( 'jetpack_send_db_checksum' )) {
+			wp_schedule_event( time(), 'hourly', 'jetpack_send_db_checksum' );
```

Comment:
It's going to be pretty dang fast, I think. Still, might be worth doing some performance tests on really large DB's. Do we have some sample "really large WP DB's" to play with? What was that post generator plugin you (or @lezama) were using?